### PR TITLE
Add ppx_jsobject_conv v0.0.2

### PR DESCRIPTION
**Pull request by little-arhat**

### CODE REVIEW

File: packages/ppx_jsobject_conv/ppx_jsobject_conv.0.0.2/opam [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+name: "ppx_jsobject_conv"
+version: "0.0.2"
+maintainer: "Roma Sokolov <sokolov.r.v@gmail.com>"
+authors: [ "Roma Sokolov <sokolov.r.v@gmail.com>" ]
+license: "MIT"
+homepage: "https://github.com/little-arhat/ppx_jsobject_conv"
+doc: "http://little-arhat.github.io/ppx_jsobject_conv"
+bug-reports: "https://github.com/little-arhat/ppx_jsobject_conv/issues"
+dev-repo: "git://github.com/little-arhat/ppx_jsobject_conv.git"
+tags: [ "syntax" "jsoo" "javascrpit" ]
```
Comment (dsheets) [inline packages/ppx_jsobject_conv/ppx_jsobject_conv.0.0.2/opam]: s/javascrpit/javascript/?

File: packages/ppx_jsobject_conv/ppx_jsobject_conv.0.0.2/opam [hunk 2]
```diff
@@ -0,0 +1,25 @@
+opam-version: "1.2"
+name: "ppx_jsobject_conv"
+version: "0.0.2"
+maintainer: "Roma Sokolov <sokolov.r.v@gmail.com>"
+authors: [ "Roma Sokolov <sokolov.r.v@gmail.com>" ]
+license: "MIT"
+homepage: "https://github.com/little-arhat/ppx_jsobject_conv"
+doc: "http://little-arhat.github.io/ppx_jsobject_conv"
```
Comment (yallop) [inline packages/ppx_jsobject_conv/ppx_jsobject_conv.0.0.2/opam]: There's currently nothing at the `doc` url.

File: packages/ppx_jsobject_conv/ppx_jsobject_conv.0.0.2/opam (hunk 1 above)
Comment (little-arhat) [inline packages/ppx_jsobject_conv/ppx_jsobject_conv.0.0.2/opam]: @dsheets good catch, thanks!

File: packages/ppx_jsobject_conv/ppx_jsobject_conv.0.0.2/opam (hunk 2 above)
Comment (little-arhat) [inline packages/ppx_jsobject_conv/ppx_jsobject_conv.0.0.2/opam]: @yallop bitten by copy-paste, sorry. Thanks!

File: packages/ppx_jsobject_conv/ppx_jsobject_conv.0.0.2/opam [hunk 3]
```diff
@@ -0,0 +1,24 @@
+opam-version: "1.2"
+name: "ppx_jsobject_conv"
+version: "0.0.3"
```
Comment (dsheets) [inline packages/ppx_jsobject_conv/ppx_jsobject_conv.0.0.2/opam]: Please remove the `name` and `version` fields.

**Pull request merged**